### PR TITLE
initial commit

### PR DESCRIPTION
**Pull request by rrogowski**
# Summary
initial commit

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,3 @@
+*.iso
+*.o
+iso/boot/kern
```
Comment (valakuzhyk) [inline .gitignore]: Do you want to just ignore everything in iso?

**Review by valakuzhyk — CHANGES_REQUESTED** [review]
Testing the review functionality

File: .gitignore (hunk 1 above)
Comment (rrogowski) [reply .gitignore]: We still need the iso/boot/grub/grub.cfg file. I'll ignore everything except this file!

**Comment by rrogowski** [comment]
ready for round 2!

**Review by valakuzhyk — APPROVED** [review]

**Pull request merged**